src/lightning_app/core/work.py

```diff
@@ -45,6 +45,7 @@ def __init__(
         cloud_build_config: Optional[BuildConfig] = None,
         cloud_compute: Optional[CloudCompute] = None,
         run_once: Optional[bool] = None,  # TODO: Remove run_once
+        drives: Optional[List[Drive]] = None,
```

Comment:
There is no need for this.

Reply:
We simply need to list the drives attached to the work once the users call work.run.

Reply:
I agree. The drives is something the user manages by named attributes. Having them in a list is not useful IMO.

Reply:
Indeed. Didn't have the best read on the API, implementing the change now ;)

Reply:
removed